dll/win32/imageres/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_definitions(-D_WINE)
+
+file(GLOB_RECURSE imageres_rc_deps res/*.*)
+add_rc_deps(imageres.rc ${imageres_rc_deps})
+
+list(APPEND SOURCE imageres.c)
+
+add_library(imageres MODULE
+    ${SOURCE}
+    imageres.rc
+    ${CMAKE_CURRENT_BINARY_DIR}/imageres.def)
+
+set_module_type(imageres win32dll UNICODE)
```

Comment:
If you use `ENTRYPOINT 0` you can completely drop the `.c` file

Reply:
See for example https://git.reactos.org/?p=reactos.git;a=blob;f=dll/nls/normaliz/CMakeLists.txt;hb=ea8a6d6fb78629eab0d36291a24716623f3a05b8#l15

Reply:
OK.